docs/content/docs/dev/datastream/sources.md

```diff
@@ -131,12 +131,12 @@ class MySplitEnumerator implements SplitEnumerator<MySplit> {
     @Override
     public void start() {
         ...
-        enumContext.callAsync(this::discoverSplits, splits -> {
-            Map<Integer, List<MockSourceSplit>> assignments = new HashMap<>();
+        enumContext.callAsync(this::discoverSplits, (splits, thrown) -> {
```

Comment:
Can we make it more obvious where the enumContext comes from? Like adding a constructor and field.

Reply:
Done!